utils.go

```diff
@@ -64,6 +66,30 @@ func DecodeLabel(image imgutil.Image, label string, v interface{}) error {
 	return nil
 }
 
+func ResolveRunImage(stackMD platform.StackMetadata, destinationImageRef string) (string, error) {
+	runImageRef := stackMD.RunImage.Image
```

Comment:
Is this in danger of throwing a `nil` panic?

Reply:
I don't think it is possible to panic here. `stackMD` is not a pointer and the definition [here](https://github.com/buildpacks/lifecycle/blob/a7428a55c2a14d8a37e84285b95dc63192e3264e/platform/files.go#L208) has `Runimage` as `StackRunImageMetadata` - also not a pointer. It should always be `""` in the worst case.

Reply:
Answer - no, because of https://github.com/buildpacks/lifecycle/blob/162090f6255b289c36fcfac8a884a8e8c945c82e/platform/files.go#L208 (no omit empty)